tests/baselines/reference/privateNameJsBadAssignment.errors.txt

```diff
@@ -0,0 +1,30 @@
+tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment.js(1,9): error TS2339: Property '#nope' does not exist on type 'typeof import("tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment")'.
+tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment.js(6,3): error TS2339: Property '#foo' does not exist on type 'typeof B'.
+tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment.js(11,17): error TS2339: Property '#bar' does not exist on type 'typeof import("tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment")'.
+tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment.js(12,14): error TS2339: Property '#foo' does not exist on type 'C'.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment.js (4 errors) ====
+    exports.#nope = 1;           // Error (outside class body)
+            ~~~~~
+!!! error TS2339: Property '#nope' does not exist on type 'typeof import("tests/cases/conformance/classes/members/privateNames/privaateNameJsBadAssignment")'.
+    function A() { }
+    A.prototype.#no = 2;         // Error (outside class body)

```

Comment:
this looks like the JS test I requested above. It's better than the TS version, although `#no` is still not an error like it should be.

Reply:
Function.prototype is type any in JS too.

Reply:
`Function.prototype: any` in JS too.